### DISCUSSION TITLE
Error installing on a MAUI app

### DISCUSSION THREAD

**Discussion by AngeloLazzari** [thudugala/Plugin.LocalNotification#512] [Q&A] [answered]
Hi, I was installing your plugin in our app but I have this error:
NU1608: Detected package version outside of dependency constraint: Xamarin.AndroidX.Lifecycle.Process 2.6.2.3 requires Xamarin.AndroidX.Lifecycle.Runtime (>= 2.6.2.3 && < 2.6.3) but version Xamarin.AndroidX.Lifecycle.Runtime 2.7.0.4 was resolved.

I don't want to install xamarin stuff on our MAUI app, can you help me pleasE? thank you very much!!!

**Comment by thudugala** [comment, marked as answer]
@AngeloLazzari 

Please have a look at https://github.com/thudugala/Plugin.LocalNotification/releases/tag/v11.1.4

Even though it is .Net Maui, all the back libraries are still called Xamarin by Microsoft. 

![image](https://github.com/user-attachments/assets/ffe66fc4-a946-440f-8dd4-6c1c2ec82afe)


![image](https://github.com/user-attachments/assets/d77e96f1-6a34-44c3-aba3-7f3c72c941f0)

**Reply by AngeloLazzari** [reply]
Thank you VERY MUCH!!! @thudugala
